data/entities.json5

```diff
@@ -9508,7 +9508,7 @@
   {
     "name": "Urban Airship",
     "categories": ["ad"],
-    "domains": ["*.urbanairship.com"]
+    "domains": ["*.urbanairship.com", "aswpsdkus.com"]
```

Comment:
Wildcarding *.aswpsdkus.com will protect against any changes.

Reply:
Thanks @zeman. Updated. Also, made following changes - 1)  "Urban Airship" got re-branded as just "Airship" recently so updated the name 2) I think "ad" was wrong category for this third party, "marketing" is more suited.